sysboundary/sysboundarycondition.cpp

```diff
@@ -785,48 +785,55 @@ namespace SBC {
       return flowtoCellsBlock;
    }
    
-   Real SysBoundaryCondition::fieldBoundaryCopyFromExistingFaceNbrMagneticField(
+   Real SysBoundaryCondition::fieldBoundaryCopyFromSolvingNbrMagneticField(
       FsGrid< std::array<Real, fsgrids::bfield::N_BFIELD>, 2> & perBGrid,
```

Comment:
This could also get renaming perBGrid to bGrid but as it's not an indentical call to the previous it's not as important.